test/common/fixtures.go

```diff
@@ -42,29 +41,30 @@ func TypeConfigFixturesOrDie(tl TestLogger) map[string]*unstructured.Unstructure
 }
 
 func typeConfigFixtures() (map[string]*unstructured.Unstructured, error) {
-	path := fixturePath()
-	files, err := ioutil.ReadDir(path)
-	if err != nil {
-		return nil, errors.Wrapf(err, "Error reading fixture from path %q", path)
-	}
-
 	fixtures := make(map[string]*unstructured.Unstructured)
-	suffix := ".yaml"
-	for _, file := range files {
-		if !strings.HasSuffix(file.Name(), suffix) {
-			continue
-		}
-
-		typeConfigName := strings.TrimSuffix(file.Name(), suffix)
-
-		filename := filepath.Join(path, file.Name())
+	for _, file := range AssetNames() {
 		fixture := &unstructured.Unstructured{}
-		err := kfenable.DecodeYAMLFromFile(filename, fixture)
+		typeConfigName, err := decodeYamlFromBindata(file, "test/common/fixtures/", fixture)
 		if err != nil {
-			return nil, errors.Wrapf(err, "Error reading fixture for %q", typeConfigName)
+			return nil, errors.Wrap(err, "Error reading a fixture")
+		}
+		if len(typeConfigName) != 0 {
+			fixtures[typeConfigName] = fixture
 		}
-		fixtures[typeConfigName] = fixture
 	}
 
 	return fixtures, nil
 }
+
+func decodeYamlFromBindata(filename, prefix string, obj interface{}) (string, error) {
```

Comment:
Please be consistent in use of casing of acronyms (e.g. `decodeYAMLFromBindata` to be consistent with `DecodeYAML`).

Reply:
Fixed